File: WorldFeed/Server/src/WorldFeed.Streams/Model/AccountActivity/AccountActivityFavouriteEventDTO.cs
namespace WorldFeed.Streams.Model.AccountActivity
{
    using Newtonsoft.Json;

    using WorldFeed.Common.Public.Models.Interfaces.DTO;

    public class AccountActivityFavoriteEventDTO
    {
        public string Id { get; set; }

        [JsonProperty("user")]
        public IUserDTO User { get; set; }

        [JsonProperty("favorited_status")]
        public ITweetDTO FavoritedTweet { get; set; }
    }
}

File: Sharebook/Client/src/app/streams/Model/AccountActivity/AccountActivityFavouriteEventDTO.ts
import {IUserDTO} from "../../../core/Public/Models/Interfaces/DTO/IUserDTO";
import {ITweetDTO} from "../../../core/Public/Models/Interfaces/DTO/ITweetDTO";

export class AccountActivityFavoriteEventDTO {
  public Id: string;

  // [JsonProperty("user")]
  public User: IUserDTO;

  // [JsonProperty("favorited_status")]
  public FavoritedTweet: ITweetDTO;
}
